### PR TITLE
Switch to sync harvest backend for maaf

### PR DESCRIPTION
**Pull request by ThibaudDauce**
See https://github.com/opendatateam/udata/pull/3030

### CODE REVIEW

File: requirements/udata.in [hunk 1]
```diff
@@ -1 +1 @@
-udata@git+https://github.com/opendatateam/udata.git
+udata@git+https://github.com/opendatateam/udata@sync_harvest_backend
```
Comment (maudetes) [inline requirements/udata.in:1]: Don't forget to rollback here :)

File: requirements/udata.pip [hunk 2]
```diff
@@ -39,7 +39,7 @@ bleach==3.3.1
 blinker==1.4
     # via
     #   flask-mail
-    #   flask-navigation
+    #   flask-navigation-temp-python-311
```
Comment (maudetes) [inline requirements/udata.pip:42]: Why wasn't the modification done sooner? :thinking:

**Review by maudetes — APPROVED** [review]
Thank you! :pray:

File: requirements/udata.pip (hunk 2 above)
Comment (ThibaudDauce) [reply requirements/udata.pip]: Don't know… I reverted the change…

**Pull request merged**